app/src/app/components/contents/downloads/DownloadButton.tsx

```diff
@@ -11,21 +11,31 @@ interface DownloadButtonProps {
 
 export const DownloadButton = ({ file, packetId }: DownloadButtonProps) => {
   const [error, setError] = useState("");
+  const [downloading, setDownloading] = useState(false);
```

Comment:
Since there's now also FileGroupDownloadButton, maybe this one should now be called "FileDownloadButton" or similar

Reply:
Will do in https://github.com/mrc-ide/packit/pull/176